cc_purkukartoitus_service/src/app.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Commands:

```

Comment:
instructions could be moved to README

Reply:
And readme moved from root to service folder.